components/Code.tsx

```diff
@@ -39,39 +39,35 @@ const Code: React.FC = ({ children }) => {
             setCopy(false);
         }, 2000);
     };
-    const [hovered, setHovered] = React.useState(false);
     const [copy, setCopy] = React.useState(false);
-    const onEnter = () => {
-        setHovered(true);
-    };
+
     const onExit = () => {
-        setHovered(false);
         setCopy(false);
     };
     return (
-        <div className="code-block" onMouseEnter={onEnter} onMouseLeave={onExit}>
-            {hovered && !copy ? (
+        <div className="code-block" onMouseLeave={onExit}>
```

Comment:
we can remove `onExit` too 

Why to change back to copy when cursor is out of exist

This is not a pattern other docs follow.